utils/install_dag.sh

```diff
@@ -33,6 +53,11 @@ AIRFLOW_DAG_PATH=${AIRFLOW_HOME}/dags/${IMAGE}
 CONTAINER_DAG_PATH=/dags
 POST_INSTALL=${AIRFLOW_DAG_PATH}/post_install.sh
 
+
+while ! wait_for_docker_daemon; do
```

Comment:
@smpiano as I mentioned in slack what I mentioned were two options that achieved the same. Youapplied both solutions at once. 
Change:
```
while ! wait_for_docker_daemon; do
  echo
done
```
with just
```
wait_for_docker_daemon
```